orm/insert.go

```diff
@@ -134,12 +134,13 @@ func (q *insertQuery) appendValues(b []byte, fields []*Field, strct reflect.Valu
 			continue
 		}
 
-		if q.placeholder {
+		switch {
+		case q.placeholder:
 			b = append(b, '?')
-		} else if (f.Default != "" || f.OmitZero()) && f.IsZeroValue(strct) {
+		case f.Default != "" || f.OmitZero() && f.IsZeroValue(strct):
```

Comment:
It does not look like an equivalent replacement.

Reply:
Yeah, you are right. I forgot the brackets.

Reply:
I submitted a pull request ( #1193 ) fixing this issue. 
Thanks very much for finding this!